### ISSUE TITLE
HumanEva erroneous annotation

### ISSUE THREAD

**Issue by isarandi** [facebookresearch/VideoPose3D#29]
I ran the code in this repo to generate the 15-joint skeleton for the HumanEva-I dataset. I notice that the head joint of subject S3 in Walking_1 is wrong for the most of the video (the whole training part and some of the validation) and it is on (in front of) the chest and not on the head.

Here is the skeleton I get:
![screenshot from 2019-02-04 15-10-46](https://user-images.githubusercontent.com/6182426/52213410-54d21800-288f-11e9-95fb-de96333f9ec0.png)

Did you deal with this in any way or did you just evaluate the method against the wrong head position in the validation set?

**Comment by dariopavllo** [comment]
Hi,

HumanEva is in general full of corrupted frames, which are due to sensor dropout and (probably) bad calibration. These errors are already present at the source, and there is no way to correct them automatically.

The original MATLAB implementation of HumanEva has some scripts to detect frames that are totally unusable (e.g. due to missing joints), and we use them to split the videos into valid chunks. Errors that are undetectable (like the one you mentioned) are unfortunately included in the training/test set. Since we had to compare to previous work, we did not want to bias our quantitative results by changing the test set manually.

We don't know how other approaches deal with this problem (they didn't release HumanEva code), but my assumption is that they use the full test set since they don't specify otherwise. Anyway, as HumanEva is rather small, Human3.6M results are more significant.
